### ISSUE TITLE
Exception when using semver-date-time

### ISSUE THREAD

**Issue by arichiardi** [degree9/boot-semver#19]
I think I found some bug maybe, or maybe I cannot use it like this...I don't have too much time now to investigate:

```
$ boot version -n -b semver-date-time
             clojure.lang.ExceptionInfo: clojure.lang.ArityException: Wrong number of args (0) passed to: core/semver-date
    data: {:file "/tmp/boot.user7467330014828633277.clj", :line 31}
java.util.concurrent.ExecutionException: clojure.lang.ArityException: Wrong number of args (0) passed to: core/semver-date
            clojure.lang.ArityException: Wrong number of args (0) passed to: core/semver-date
                                ...                
  boot-semver.core/semver-date-time  core.clj:   48
                                ...                
 boot-semver.core/update-version/fn  core.clj:   53
clojure.core/merge-with/merge-entry  core.clj: 2932
               clojure.core/reduce1  core.clj:  909
     clojure.core/merge-with/merge2  core.clj: 2935
               clojure.core/reduce1  core.clj:  909
               clojure.core/reduce1  core.clj:  900
            clojure.core/merge-with  core.clj: 2936
                                ...                
    boot-semver.core/update-version  core.clj:   53
       boot-semver.core/eval2449/fn  core.clj:   84
                                ...                
                 clojure.core/apply  core.clj:  630
          boot.core/construct-tasks  core.clj:  760
                                ...                
                 clojure.core/apply  core.clj:  630
                  boot.core/boot/fn  core.clj:  805
clojure.core/binding-conveyor-fn/fn  core.clj: 1916
```

**Comment by burn2delete** [comment]
@arichiardi fixed! Please use version 1.2.1

**Comment by arichiardi** [comment]
Efficient! :)
